src/steamship_langchain/llms/openai.py

```diff
@@ -1,63 +1,106 @@
 import hashlib
+import json
 import logging
-from typing import Any, List, Mapping, Optional
+from collections import defaultdict
+from typing import Any, Dict, Generator, List, Mapping, Optional
 
-import tiktoken
-from langchain.llms.base import BaseLLM, Generation, LLMResult
-from pydantic import Field
+from langchain.llms.base import Generation, LLMResult
+from langchain.llms.openai import BaseOpenAI
+from pydantic import root_validator
 from steamship import Block, File, Steamship, SteamshipError
 from steamship.data import TagKind, TagValueKey
 
-
-class OpenAI(BaseLLM):
+PLUGIN_HANDLE: str = "openai"
```

Comment:
this needs to be `gpt-3`, correct?

Reply:
Yes, was deploying a new version of `gpt-3` with the `openai` handle for testing purposes.